blocks/address/turbo.json

```diff
@@ -0,0 +1,28 @@
+{
+  "extends": ["//"],
```

Comment:
I just noticed this, what does the "//" mean here? I guess it means root, but why is it '//'? :smile:

Reply:
No idea why 😅 
https://turbo.build/repo/docs/core-concepts/monorepos/configuring-workspaces